builtin/aws/ecs/platform.go

```diff
@@ -1942,6 +1947,15 @@ func (p *Platform) resourceExternalSecurityGroupsCreate(
 	s := sg.Add("Initiating creation of external security group named %s", name)
 	defer s.Abort()
 
+	if p.config.ALB.SecurityGroupIDs != nil {
+		s.Update("Using specified security group IDs")
```

Comment:
```suggestion
		s.Update("Using specified ALB security group IDs")
```
Or other variation to give more specific log output.